repo.json

```diff
@@ -1,12 +1,12 @@
 {
   "catalogs": {
     "library": {
-      "url": "https://github.com/rancher/rancher-catalog",
```

Comment:
For future reference, its good practice to separate unrelated changes into separate commits. Makes rolling back easier when it is needed, this change to the repo should ideally be its own commit. No need to change it though, just a heads up.